### DISCUSSION TITLE
What's the right version of kustomize binary to match flux v2?

### DISCUSSION THREAD

**Discussion by jam01** [fluxcd/flux2#1304] [Q&A]
Hey guys, we're seeing some errors from the kustomize binary whereas the flux operator works fine one some specific setups. We're trying to figure out what's the specific binary version we should be using to match what flux does

**Comment by jam01** [comment]
Specifically there are security errors around referencing a resource on a parent directory

**Comment by stefanprodan** [comment]
Kustomize version here https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/.github/actions/tools/entrypoint.sh#L6
Build args here https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/scripts/validate.sh#L33
